File: w03/d03/Wake_Lankard/squares/kitten.rb
class Kitten
  def initialize
    @height = rand(400)+50
    @width = rand(400)+50
  end

  def to_s
    "http://placekitten.com/#{@height}/#{@width}"
  end

  def random
    @height = rand(500)+50
    @width = rand(500)+50
    "http://placekitten.com/#{@height}/#{@width}"
  end
end

File: w08/d02/Jonathan_Gean/the_definitive_kinder/app/assets/javascripts/kitten.js
function randomURL(){
  var width = Math.floor(Math.random()*200+100);
  var height = Math.floor(Math.random()*200+100);
  return "http://placekitten.com/" + width + "/" + height;

}

function KittenModel(url){
  this.url = url || randomUrl();
}

function KittenView(model){
  this.model = model;
  this.el    = undefined;
}

KittenView.prototype.render = function (){
  var img = $('<img>').attr('src', this.model.url)
                      .addClass('kitten');
  this.el = img;

}

function KittenCollection(){
  this.kittens = {}
}
